lib/spring/client/binstub.rb

```diff
@@ -13,8 +13,9 @@ class Binstub < Command
       # should cause the "unsprung" version of the command to run.
       LOADER = <<CODE
 begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError
+  load spring_loader = File.expand_path('../spring', __FILE__)
```

Comment:
Good catch, both `load File.expand_path('../spring', __FILE__)` and `load spring_loader` match the regex `/load .*spring/` in `Spring::Binstub::Item#add`, so it's OK.